_drafts/oryx/boilerplate.md

```diff
@@ -0,0 +1,139 @@
+# Understanding Boilerplate in Oryx
+
+"How to Create Maintainable and Upgradeable Applications"
+
+**TLTR:**
+
+> Oryx boilerplate is provided at https://github.com/spryker/composable-frontend.
+
+## Introduction
+
+Boilerplate refers to the _template_ code that is used to generate an application code that can be further customized. Changing boilerplate is convenient at first sight as the code is at hand, generated in your project repository. However, when upgrading to newer versions of the original code, it becomes challenging. If you've customize the boilerplate code and want to reapply a new version of the boilerplate, you have to merge the customizations with the boilerplate during every update to a new version of the application. This is a time-consuming and error-prone process that can slow down your development process and increase the risk of bugs.
+
+## How we prevent boilerplate code
+
+In Oryx, we use a few tactics to prevent boilerplate code. These tactics include:
+
+1. Bootstrap the application from npm packages rather than source.
+2. Expose an function to setup the [app orchestrator](./app-orchestrator.md) conveniently.
+3. Provide [presets](./presets.md) for the standard application setup, including the feature sets and UI themes.
+4. Provide the source code in a [public repository](https://github.com/spryker/oryx) to enable developers to read the source code.
+5. Provide configurable component and business logic.
```

Comment:
`configurable component` and `business logic` - should be links?

Reply:
once we have those docs, yes.